luigi_pipeline/seqr_gcnv_loading.py

```diff
@@ -37,6 +98,7 @@ def generate_callstats(self, mt, *args, **kwargs):
 
     def import_dataset(self):
         ht = hl.import_table(self.source_paths[0], impute=True, min_partitions=500)
```

Comment:
I think it would be better to pass `types` in here instead of imputing types and then validating. For anything you don't specify, hail will default them to strings, which for fields we never use is acceptable behavior so you will only need to pass in the fields we care about

Reply:
👍 makes sense!

Reply:
From what I'm seeing hail does the validation for us.  If hail can't parse as the type specified it'll throw an exception.